### ISSUE TITLE
Accessibility: tooltips not read to screen readers

### ISSUE THREAD

**Issue by paulGeoghegan** [shoelace-style/webawesome#1218]
### Describe the bug
Currently when you hover it does not read tooltips to screen readers. You have to arrow up in order to read them but obviously a user wouldn't know that it's there.

### To Reproduce

1. Go to [wa-tooltip docs](https://webawesome.com/docs/components/tooltip/)
2. focus on the first tooltip example while using a screen reader
3. Nothing is announced
4. Arrow up
5. Tooltip is read.

### Browser / OS
 - OS: Windows
 - Browser: Firefox
 - Browser version: 140.0.4

**Comment by paulGeoghegan** [comment]
@claviska I know you added help wanted, is that because you guys aren't sure how to resolve this? If so let me know, I can maybe take a look in to it myself.

**Comment by claviska** [comment]
A concrete step forward would be helpful, or a PR. Thanks!

**Comment by paulGeoghegan** [comment]
@claviska I've been looking in to this. I have played around with it a bit.
So is the tooltip in the shadow dom?
I also think we may need to use an aria live region or something here as I have tried with multiple screen readers and none of them read it. I'm not super happy with an aria-live work around but I re-checked aria-described by and that doesn't work either.
I also tried using visibility hidden instead of display none in case that was causing issues but no luck.

**Comment by claviska** [comment]
Web Awesome tooltips were redesigned to follow this pattern:

https://www.w3.org/WAI/ARIA/apg/patterns/tooltip/

Which admittedly is still under development, but it's the best "official" guidance I've been able to find in years.

> NOTE: This design pattern is work in progress; it does not yet have task force consensus. Progress and discussions are captured in [issue 128.](https://github.com/w3c/aria-practices/issues/128)

That said, the tooltips are reading for me in VoiceOver in Chrome, Safari, and Firefox. There have been long discussions about tooltips prior to this rework in the Shoelace repo, which [previously used aria-live](https://github.com/shoelace-style/shoelace/blob/1ef287c3c13c794e8e5ce4fddf29511510c8c65b/src/components/tooltip/tooltip.component.ts#L257-L262). Unfortunately, this has led to [other issues](https://github.com/shoelace-style/shoelace/issues/2406).

Here are just some of the issues and discussions from the past, a few of which are still open:

- https://github.com/shoelace-style/shoelace/issues/219
- https://github.com/shoelace-style/shoelace/issues/1712
- https://github.com/shoelace-style/shoelace/issues/1734
- https://github.com/shoelace-style/shoelace/issues/2406

That said, I'm reluctant to change the pattern we're using because, historically, tooltip accessibility has been a game of whack-a-mole. We fix something in one screen reader, it breaks in another. We have one expert say A and then another says B. The difference is usually what browser + screen reader they're using.

I'm really not sure how to solve this for every screen reader and browser combination, but using live regions sounds like a big step back into the problems we had (and still have to be fair) with Shoelace.

**Comment by paulGeoghegan** [comment]
I do agree hence why I said i'm not happy with that as a solution. I'm on
windows and have tried multiple browsers and screen readers.
I'll try have more of a think and a look to see if I can find or think of
something that might work.

**Comment by claviska** [comment]
Thanks. Let me know what you come up with. Until then, this is "working per APG guidelines" but if there's a concrete way to improve on tooltip accessibility, we're very happy to work on it!
